src/matlab.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 The MathWorks, Inc.
+
+import * as core from "@actions/core";
+import * as http from "@actions/http-client";
+import * as io from "@actions/io";
+import * as tc from "@actions/tool-cache";
+import * as fs from "fs";
+import properties from "./properties.json";
+import * as script from "./script";
+
+export interface Version {
+    release: string;
+    semantic: string;
+}
+
+interface MATLABVersionInfo {
```

Comment:
Should this be `MATLABReleaseInfo` for consistency with the url? Or, if not, should the url be `matlabVersionInfoUrl`?

Reply:
Updated to `MATLABReleaseInfo`. I could see either being technically correct, but the user specifies the "release" parameter so maybe we should stick with that.